src/lib.rs

```diff
@@ -648,6 +648,33 @@ impl Builder {
         self
     }
 
+    /// Add multiple input C/C++ headers.
+    pub fn headers<T: AsRef<str>>(
```

Comment:
Is there a reason to prefer `<T: AsRef<str>>` here rather than `<T: Into<String>>` which matches the existing `header` function?

I am imagining that `headers` might be called with `[String]`, and for each `String`, `.as_ref()` would return `&str`, which is then made into a new `String` in the `header` function. Would not using the same `T` as `header` would avoid this extra allocation?

The same argument would apply to `include` and `includes`, too.

Reply:
I'm not sure to be honest, I've seen both `AsRef<str>` and `Into<String>` being used in the API so I don't have strong opinions against either of those.

Reply:
I'm not overly worried about double-allocating a string here - it's likely that the optimizer will see it, and even if it doesn't, we're not in a hot path - our user will call us once and then get on with their lives.

Changed anyway